Automattic/wp-calypso
# Editor: Disable publish/update button when revisions list is visible

**Note**: This PR is based on the `update/move-nested-sidebar-state-to-redux` branch which is up for review in #19294. This PR will point at that branch once it's ready.

### Summary
Having two primary buttons so close to each other ('Update' in EditorGroundControl and 'Load Revision...' in the revisions sidebar could lead to folks clicking the wrong button ('publish') with the intent of directly publishing their selected revision.
To avoid this confusion this PR disables the `publish`/`update` button while revisions are open.

### Testing
- Go to the calypso.live branch, or build locally on this branch.
- Go to the editor view with a post that has had revisions.
- Open the revisions list sidebar by either clicking 'History' in the header or the 'xx revisions' button in the post sidebar
- Note that the `publish`/`update` is greyed out and not actionable.
- Close the revisions list by clicking history once more or the 'x' button
- Note that the `publish`/`update` is now enabled and actionable.
